science/projects/soroush_heidary/01-main-content/Main-Content.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    clf = RandomForestClassifier(max_depth=9, random_state=0)
</pre><p>As a suggestion, </p><p>you can test different parameters for the model using GRID SEARCH or Optuna and select the best parameter.</p><p>https://github.com/optuna/optuna</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>